custom.conf.template

```diff
@@ -1,6 +1,6 @@
 [Labels]
 # For each parameter, either provide a list of comma-separated labels, or leave blank if not using
-required-labels-any:
+required-labels-any:sample-required-label
```

Comment:
what's this for?

Reply:
we now check on boot if there are any label set at all. Without this, the tests fail. I suppose I could set up a test conf and use that in the tests instead?

Reply:
Ah gotcha. My hesitation is just atht this is the public-facing template, so it looks a little funny. If it's easier to have something here, I might prefer going all-out in terms of an example, and doing something like:

```
required-labels-any: one-of-these,is,required
required-labels-all: all-of-these,are,required
banned-labels: these,are,banned
```
thoughts?